python/ray/utils.py

```diff
@@ -21,12 +21,15 @@
 import ray.ray_constants as ray_constants
 
 
-def _random_string():
+def _random_string(length=ray_constants.ID_SIZE):
```

Comment:
maybe we should deprecate this function, just use `ObjectID.random()`.

Reply:
@raulchen I have discussed this with @pcmoritz . There will be some problem to use this in multi-thread cases.It may generated the same ObjectID.

Reply:
https://github.com/ray-project/ray/pull/3674#issuecomment-450989756

Reply:
I see. But I think we can move the same code to Cython's `ObjectID.random` to avoid that problem.